apache/superset
# chore(de-translations): Add and fix German translations

### SUMMARY
This PR adds missing and improves some German translations, based on current `POT` file.
 
### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS
1. Enable German translations by adding `"de": {"flag": "de", "name": "Deutsch"},` to the enabled `LANGUAGES` (I'd eve suggest to add this now per default, as it should be fully translated)
2. start superset
3. Switch to German translations and check, if all messages are translated and meaningful

### ADDITIONAL INFORMATION
